index.js

```diff
@@ -2,9 +2,10 @@ const api = require('./sf_api')
 const fsUtils = require('./fs_utils')
 const fs = require('fs');
 
+const RECONCILIATOIN_FIELDS_TO_SYNC = ["name_nl", "name_fr", "name_en", "auto_hide_formula", "text_configuration", "virtual_account_number", "reconciliation_type", "public", "allow_duplicate_reconciliations", "is_active", "tests"]
+
 createNewTemplateFolder = async function (handle) {
-  const relativePath = `./${handle}`
-  const emptyCallback = () => {}
+  const relativePath = `./data/${handle}`
```

Comment:
Why the path was changed to `data`?

Reply:
We need to group everything in a folder to add it to .gitignore. Other solution is to prefix folders.
Since users will play with the templates code and also the sf-toolkit code, I thought it's usefull to have the data code is ignored

Reply:
@ihcene-silverfin one of the requests from PST was to subfolder everything under `reconciliations_texts`/`shared_parts`/`account_detail_templates`. I believe this requests conflicts with adding `data` superfolder

Reply:
I will take that into account thank you